### PR TITLE
Make selection lines visible and fix safezone offset

### LINKED ISSUE #1545: Lasso selection coordinates inaccurate
## Bug Description

Lasso selection coordinates seem to be inaccurate and not following the cursor.

## Expected Behaviour
Lasso selection should be displayed based on the cursor coordinates and movement.

## Steps to Reproduce
1. Try lasso selecting
2. See the outline being created inaccurately

## Screenshots
lasso

---

_Do not alter or remove anything below. The following sections will be managed by moderators only._

### LINKED ISSUE #1787: Element edges/borders incorrect when part of the element is outside of canvas
## Bug Description

If I create an element (shape, images) and drag them down to below the bottom edge of the canvas their selection area is incorrectly placed as starting from the bottom to the canvas up, covering other parts of the canvas where the shape/image is not. 

## Steps to Reproduce

1. Add shape
2. Drag shape partially outside of canvas
3. select element

## Screenshots

Video: 
[Odd selection3.mov.zip](https://github.com/google/web-stories-wp/files/4629531/Odd.selection3.mov.zip)

---

_Do not alter or remove anything below. The following sections will be managed by moderators only._

### PR DESCRIPTION
**Pull request by dvoytenko**
## Summary

Fixes the following issues:
1. The selection lines are now visible when they overflow the page area. However, they are placed behind page menu and other nav controls.
2. The safezone offset is fixed when an overflowing element is selected.

The best I can tell the bug arises due to Chrome bug. Simply doing `$0.style.position = 'relative'; $0.style.position = 'absolute'` (i.e. resetting the position style) fixes the issue.

## Relevant Technical Choices

The solution is to move the moveable and selection lines to the fullbleed zone, out of safezone.

## To-do

- [x] Check how this behaves with lasso
- [x] Karma tests

## User-facing changes

The selection lines are visible for overflowing elements.

## Testing Instructions

See #1787

---

Fixes #1787
Fixes #1545

### CODE REVIEW

**Review by wassgha — APPROVED** [review]
Works well, I'll reevaluate whether the other changes on #2269 are needed or not

File: assets/src/edit-story/components/canvas/framesLayer.js [hunk 1]
```diff
@@ -89,6 +89,7 @@ function FramesLayer() {
       tabIndex="-1"
     >
       <FramesPageArea
+        showOverflow
```
Comment (merapi) [inline assets/src/edit-story/components/canvas/framesLayer.js]: Where should we put it? https://github.com/google/web-stories-wp/commit/d533bec7f6adc8c411391acf48c3604af113a762

Comment (dvoytenko) [reply assets/src/edit-story/components/canvas/framesLayer.js]: Either way, imho. Let me know what you'd prefer. But I generally like them in markup when the meaning is "logical state". And for layers, the overflow could be considering a logical state.

Comment (merapi) [reply assets/src/edit-story/components/canvas/framesLayer.js]: After I saw [this](https://github.com/google/web-stories-wp/blob/32042bd166b66c5fce41ff2e1204d440faaf049b/assets/src/edit-story/components/canvas/layout.js#L156) my thought was: if the value is fixed, put it there.

Comment (dvoytenko) [reply assets/src/edit-story/components/canvas/framesLayer.js]: Sounds good. Done.

**Pull request merged**